### PR TITLE
agregando ruta privada para usuarios registrados

### PR DESCRIPTION
**Pull request by ervinj1998**
En esta solicitud se crea una ruta privada en la edicion del perfil y en perfil personal, para abrirse en otro navegador tiene que registrarse primero o autenticarse.

### CODE REVIEW

**Comment by Hawaiideveloper** [comment]
What chapter is this Ervin.  Please comment below?

**Comment by ervinj1998** [comment]
Prívate router for authenticated User. - 138

**Comment by Hawaiideveloper** [comment]
okay I will leave comments.   Please answer them and we need @manzanarez83 to review

**Comment by Hawaiideveloper** [comment]
Prívate router for authenticated User. - 138

Cool

File: src/MainRouter.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import about from './core/About';
+import about from './core/About'; //no lo ocupamos por ahora.
```
Comment (Hawaiideveloper) [inline src/MainRouter.js:3]: do not change this one, this is a script that has a space for our about us section.  That will be used later.

File: src/auth/PrivateRoute.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
```
Comment (Hawaiideveloper) [inline src/auth/PrivateRoute.js:1]: this is the new file created for private routes

**Comment by Hawaiideveloper** [comment]
Chapter 138 to be merged:
Issue created at:
https://github.com/Hawaiideveloper/Social_Network_Front_end/issues/60

**Pull request merged**